lndr-backend/data/lndr-server.config.j2

```diff
@@ -31,7 +31,7 @@ db {
   name = "{{DB_NAME}}"
 }
 
-execution-address = "{{ETHEREUM_ACCOUNT}}"
+execution-private-key = "{{ETHEREUM_ACCOUNT}}"
```

Comment:
Is this value going to be the actual private key, or still the account identifier? If it's going to be the private key, let's go ahead and change the name of this configuration variable, too.

Reply:
Yep, will do.

Reply:
It will be the actual private key. In the future, it'll be the master extended private key of an HD wallet of execution keys that we rotate through.